features/FEATURE_LWIP/lwip-interface/CONTRIBUTING.md

```diff
@@ -0,0 +1,129 @@
+# Description
+
+This document describes how to update mbed OS lwip stack. Mbed OS lwip stack is a copy of
+lwip master repository. Stack is located in `FEATURE_LWIP/lwip-interface/lwip` directory.
+
+When new releases or single commits are added from lwip master repository they need to be
+fetched via mbed OS lwip fork repository. Repository is used to rename lwip source files 
+with `lwip_` prefix to make then compatible with mbed OS build system.
```

Comment:
- Please change "`lwip_` prefix" to "the `lwip_` prefix".
- Please change "then" to "them".
- Please change "mbed OS" to "the mbed OS".